src/plugins/intel_cpu/src/nodes/subgraph.cpp

```diff
@@ -22,32 +21,47 @@
 #include "snippets/lowered/pass/mark_loops.hpp"
 #include "transformations/defs.hpp"
 #include "transformations/cpu_opset/common/pass/convert_to_swish_cpu.hpp"
+
+#if defined(OPENVINO_ARCH_ARM64)
+#include "emitters/snippets/aarch64/cpu_generator.hpp"
+#include "transformations/snippets/common/pass/mul_add_to_fma.hpp"
+#include "transformations/snippets/aarch64/shape_inference.hpp"
+#else
+#include "emitters/snippets/x64/cpu_generator.hpp"
 #include "transformations/snippets/x64/pass/lowered/brgemm_blocking.hpp"
 #include "transformations/snippets/x64/pass/lowered/fuse_load_store_and_convert.hpp"
 #include "transformations/snippets/x64/pass/lowered/set_brgemm_copy_b_buffers_shape.hpp"
-#include "transformations/snippets/x64/pass/mul_add_to_fma.hpp"
+#include "transformations/snippets/common/pass/mul_add_to_fma.hpp"
 #include "transformations/snippets/x64/pass/remove_converts.hpp"
 #include "transformations/snippets/x64/pass/set_brgemm_cpu_blocking_params.hpp"
 #include "transformations/snippets/x64/pass/brgemm_to_brgemm_cpu.hpp"
 #include "transformations/snippets/x64/pass/enforce_precision.hpp"
 #include "transformations/snippets/x64/shape_inference.hpp"
+#endif
+
 #include "utils/cpu_utils.hpp"
 #include "utils/ngraph_utils.hpp"
 
 #include <algorithm>
 #include <array>
 #include <vector>
 
-#if defined(__linux__) && defined(SNIPPETS_DEBUG_CAPS)
+#if defined(__linux__) && defined(OPENVINO_ARCH_X86_64) && defined(SNIPPETS_DEBUG_CAPS)
 #include "emitters/snippets/x64/jit_segfault_detector_emitter.hpp"
 #include <signal.h>
 std::mutex err_print_lock;
 #endif
 
 using namespace dnnl::impl::utils;
 using namespace dnnl::impl::cpu;
+
+#if defined(OPENVINO_ARCH_ARM64)
+using namespace dnnl::impl::cpu::aarch64;
+using namespace Xbyak_aarch64;
+#else
 using namespace dnnl::impl::cpu::x64;
 using namespace Xbyak;
+#endif
```

Comment:
Agree! I removed these 4 usings, and because of https://github.com/openvinotoolkit/openvino/pull/23180#discussion_r1537052222, I used full namespaces for these mayisue(). Thanks Ivan!